js/views/components/Moderators.js

```diff
@@ -150,21 +150,22 @@ export default class extends baseVw {
     this.checkNotFetched();
   }
 
-  processMod(profile) {
-    // don't add profiles that are not moderators unless showInvalid is true. The ID list may have
+  processMod(data) {
+    // Don't add profiles that are not moderators unless showInvalid is true. The ID list may have
     // peerIDs that are out of date, and are no longer moderators.
-    const validMod = profile.moderator && profile.moderatorInfo;
+    // If the data comes from an error, the peerID is peerId instead.
+    const validMod = data.moderator && data.moderatorInfo;
     // if the moderator has an invalid currency, remove them from the list
     const buyerCur = app.serverConfig.cryptoCurrency;
-    const modCurs = profile.moderatorInfo && profile.moderatorInfo.acceptedCurrencies || [];
+    const modCurs = data.moderatorInfo && data.moderatorInfo.acceptedCurrencies || [];
     const validCur = modCurs.includes(buyerCur);
 
     if ((!!validMod && validCur || this.options.showInvalid)) {
-      this.moderatorsCol.add(new Moderator(profile, { parse: true }));
-      this.removeNotFetched(profile.peerID);
+      this.moderatorsCol.add(new Moderator(data, { parse: true }));
+      this.removeNotFetched(data.peerID || data.peerId);
```

Comment:
Why are two peerId variables being sent named with slightly different casing?

Reply:
@rmisio I refactored things to send both the profile data and the error data to this function, when it's a profile it's peerID, when it's an error it's peerId because the server doesn't capitalize the d.

Reply:
Would it be better to standardize it in getModeratorsByID so the variable names match when they arrive at processMod? Seems really awkward IMHO that processMod needs to be on the lookout for both.